src/main/java/com/beerjournal/breweriana/collection/CollectionController.java

```diff
@@ -24,11 +27,12 @@
             @RequestParam(value = "lacking", defaultValue = "false") boolean lacking,
             @RequestParam(value = "page", defaultValue = "0") int page,
             @RequestParam(value = "count", defaultValue = "10") int count,
-            @RequestParam(value = "name", defaultValue = "", required = false) String name,
-            @RequestParam(value = "category", defaultValue = "", required = false) String category) {
+            @RequestParam(value = "sortBy", defaultValue = "createtime") String sortBy,
+            @RequestParam(value = "sortType", defaultValue = "desc") String sortType,
+            @ApiIgnore @RequestParam(required = false) Map<String,String> filterRequestParams) {
```

Comment:
@ApiIgnore is for swagger purpose, people will no use it explicity (sadly they need to use external software i.e. postman in order to pass some query parameters but I believe that's good enough).